### PR TITLE
Add ARIA:document role

### PR DESCRIPTION
**Pull request by raych2**
# What Changed
Added ARIA:document role to update  react-snippets.code-snippets.

# Why

Todo:
- [ ] Add Semantic Version Label 
- [ ] Add tests
- [ ] Add docs

### CODE REVIEW

File: snippets/react-snippets.code-snippets [hunk 1]
```diff
@@ -89,6 +89,18 @@
...
+        "prefix": [
+            "DocumentAria"
+        ],
+        "body": [
+            "role='document'",
+            "tabIndex={0}",
+            "aria-expanded={$1}",
+            ""
+        ],
+        "description": "Add `role='document'` to application or other interactive widget role to switch to browse or read mode. Add `aria-expanded` to indicate whether the document is currently expanded or collapsed if the document element is collapsible."
```
Comment (kendallgassner) [inline snippets/react-snippets.code-snippets:102]: Can you add this to the readme too :)

Comment (raych2) [reply snippets/react-snippets.code-snippets:102]: Yes! Just added it!

**Review by kendallgassner — APPROVED** [review]
beautiful!

**Comment by kendallgassner** [comment]
closes(#15)

**Pull request merged**